shared/chat/inbox/container.js

```diff
@@ -223,7 +224,10 @@ const mapStateToProps = (state: TypedState, {isActiveRoute, routeState}: OwnProp
     isLoading: inboxGlobalUntrustedState === 'loading' || state.chat.get('inboxSyncingState') === 'syncing',
     neverLoaded: inboxGlobalUntrustedState === 'unloaded',
     _user: Constants.getYou(state),
-    inSearch: state.chat.get('inSearch'),
+    showNewConversation: state.chat.get('inSearch') ||
+      (selectedConversationIDKey &&
+        // $FlowIssue this is a string
```

Comment:
Can you also remove the `// $FlowIssue`? Otherwise flow will spit out a warning.